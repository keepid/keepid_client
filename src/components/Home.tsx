import React, { Component } from 'react';
import { Helmet } from 'react-helmet';
import { Link } from 'react-router-dom';
import HubLogo from '../static/images/hubs.svg';
import DatabaseLogo from '../static/images/database.svg';
import AidPlatLogo from '../static/images/aidplatform.svg';
import HomeForm from '../static/images/home-forms.svg';
import JobAppGraphic from '../static/images/job-app.svg';
import SignUp from '../static/images/sign-up.svg';
import SyncFiles from '../static/images/sync-files.svg';
import SecureGraphic from '../static/images/security.svg';
import ControlGraphic from '../static/images/control.svg';
import DocTransferGraphic from '../static/images/doc-transfer.svg';
import ConstantAvailabilityGraphic from '../static/images/constant-availability.svg';
import Spreadsheet from '../static/images/spreadsheet.svg';
import Access from '../static/images/access-data.svg';
import FileCloud from '../static/images/file-cloud.svg';

interface Props {
  // autoLogout is true if the user automatically logged out and was redirect to this page
  autoLogout: boolean,
  resetAutoLogout: () => void,
}

class Home extends Component<Props, {}, {}> {
  componentWillUnmount() {
    const {
      resetAutoLogout,
    } = this.props;
    resetAutoLogout();
  }

  render() {
    const {
      autoLogout,
    } = this.props;

    return (
      <div className="">
        <Helmet>
          <title>Welcome</title>
          <meta name="description" content="Keep.id" />
        </Helmet>

        {autoLogout
          ? (
            <div className="alert alert-warning" role="alert">
              You were automatically logged out and redirected to this page.
            </div>
          )
          : null}

        <div className="">
          <div className="">
            <div className="container-fluid">
              <div className="row mt-4">
                <div className="col-md-6">
                  <div className="background ml-5 p-4 rounded mb-3 pb-5">
                    <div className="page-header">
                      <span className="brand-text">
                        Safeguarding Identities of Those Experiencing Homelessness
                      </span>
                    </div>
                    <h4 className="pt-2 pb-2 brand-subtext">
                      A secure document storage platform for identification, reducing barriers in obtaining government aid, jobs, and homelessness services.
                    </h4>
                    <h2 className="mt-2 pt-2 pb-2">
                      <span className="home-subtext">For Those Experiencing Homelessness</span>
                    </h2>
                    <h5>Find a nearby registered organization that can help get you started</h5>
                    <Link to="/find-organization">
                      <button className="btn btn-primary btn-lg loginButtonBackground w-50">Find Organizations</button>
                    </Link>
                    <h2 className="mt-4">
                      <span className="home-subtext">For Organizations: Partner with Us</span>
                    </h2>
                    <h5>Start your free three-month trial to empower clients you serve</h5>
                    <Link to="/organization-signup">
                      <button className="btn btn-primary btn-lg loginButtonBackground w-50">Try Keep.id for Free</button>
                    </Link>
                  </div>
                </div>
                <div className="col-md-6 custom-vertical-center">
                  <div className="mr-5 p-4 container-home-right">
                    <div>
                      <img alt="Hubs" src={HomeForm} className="home-form-svg text-left" />
                    </div>

                  </div>
                </div>
              </div>
            </div>
            <div className="container mt-4 mb-4 pt-5 pb-5">
              <div className="row d-flex align-items-center">
                <div className="col-md-6 ">
                  <h1 className="text-center m-3 pb-5">
                    The lack of
                    {' '}
                    <span className="text-primary-theme font-weight-bold">identification</span>
                    {' '}
                    is a serious struggle for those experiencing homelessness.
                  </h1>
                </div>
                <div className="col-md-6">
                  <div className="row d-flex align-items-center">
                    <div className="col-6 text-center">
                      <h1 className="font-weight-bold statistic-text">54%</h1>
                    </div>
                    <div className="col-6">
                      <span className="statistic-subtext">Denied access to shelters or adequate housing services</span>
                    </div>
                  </div>
                  <div className="row d-flex align-items-center">
                    <div className="col-6 text-center">
                      <h1 className="font-weight-bold statistic-text">53%</h1>
                    </div>
                    <div className="col-6">
                      <span className="statistic-subtext">Denied access to food stamps</span>
                    </div>
                  </div>
                  <div className="row d-flex align-items-center">
                    <div className="col-6 text-center">
                      <h1 className="font-weight-bold statistic-text">45%</h1>
                    </div>
                    <div className="col-6">
                      <span className="statistic-subtext">Denied access to Medicaid or other medical services</span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div className="container-fluid mx-0 mt-4 mb-4 pt-5 pb-5 background">
              <div className="container">
                <div className="row text-center">
                  <div className="col-md-4 flex-column ">
                    <h1 className="font-weight-bold statistic-text">552,830</h1>
                    <span className="statistic-subtext">Homeless Population in U.S.</span>
                    <p className="text-muted pt-2 pb-2"><a href="https://www.projecthome.org/about/facts-homelessness">Source</a></p>
                  </div>
                  <div className="col-md-4 flex-column">
                    <h1 className="font-weight-bold statistic-text">5,800</h1>
                    <span className="statistic-subtext">Chronically Homeless Population in Philadelphia</span>
                    <p className="text-muted pt-2 pb-2"><a href="https://www.projecthome.org/about/facts-homelessness">Source</a></p>
                  </div>
                  <div className="col-md-4 flex-column">
                    <h1 className="font-weight-bold statistic-text">6,583</h1>
                    <span className="statistic-subtext">Youth experiencing Homelessness in Philadelphia</span>
                    <p className="text-muted pt-2 pb-2"><a href="https://www.projecthome.org/about/facts-homelessness">Source</a></p>
                  </div>
                </div>
              </div>
            </div>
            <div className="container mt-5 mb-3">
              <div className="row">
                <div className="col-md-6 custom-vertical-center">
                  <h1 className="text-center font-weight-bold m-3 pb-5">
                    We partner with aid organizations, leveraging existing resources and programming.
                  </h1>
                </div>
                <div className="col-md-6">
                  <div className="row pb-5">
                    <div className="col-md-4 mb-2">
                      <img alt="Hubs" src={HubLogo} className="home-svgs float-right" />
                    </div>
                    <div className="col-md-8 d-flex flex-column home-text">
                      <h3>Non-profit Focused</h3>
                      <span>
                        Local nonprofits against homelessness become hubs for Keep.id services
                      </span>
                    </div>
                  </div>
                  <div className="row pb-5">
                    <div className="col-md-4 mb-2">
                      <img alt="Database" src={DatabaseLogo} className="home-svgs float-right" />
                    </div>
                    <div className="col-md-8 d-flex flex-column home-text">
                      <h3>Security First</h3>
                      <span>
                        Keep.id securely stores documents and records for those experiencing homelessness
                      </span>
                    </div>
                  </div>
                  <div className="row pb-5">
                    <div className="col-md-4 mb-2">
                      <img alt="Aid platform" src={AidPlatLogo} className="home-svgs float-right" />
                    </div>
                    <div className="col-md-8 d-flex flex-column home-text">
                      <h3>Efficient and Relational</h3>
                      <span className="home-text">
                        Keep.id becomes an aid platform to streamline access to assistance programs and strengthen relationships between organizations and people
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>

            <div className="bg-secondary">
              <div className="container">
                <div className="jumbotron jumbotron-fluid bg-transparent text-center pb-2 mb-2">
                  <h1 className="display-5 text-light">Meaningful Client Journeys</h1>
                  <p className="lead text-white-50">Prioritizing the user experience and alleviating pain points.</p>
                </div>
                <div id="carouselExampleCaptions" className="carousel slide" data-ride="carousel">
                  <ol className="carousel-indicators">
                    <li data-target="#carouselExampleCaptions" data-slide-to="0" className="active" />
                    <li data-target="#carouselExampleCaptions" data-slide-to="1" />
                    <li data-target="#carouselExampleCaptions" data-slide-to="2" />
                    <li data-target="#carouselExampleCaptions" data-slide-to="3" />
                    <li data-target="#carouselExampleCaptions" data-slide-to="4" />
                  </ol>
                  <div className="carousel-inner">
                    <div className="carousel-item active">
                      <img src={SignUp} className="d-block w-100 mx-auto my-4 d-block home-svgs" alt="..." />
                      <h4 className="text-center text-light">Step #1: Registration</h4>
                      <p className="text-center text-white-50 pb-5 w-50 mx-auto">Homeless create a Keep.id account at participating nonprofits. Nonprofits then help homeless obtain missing identification.</p>
                    </div>
                    <div className="carousel-item">
                      <img src={SyncFiles} className="d-block w-100 mx-auto my-4 d-block home-svgs" alt="..." />
                      <h4 className="text-center text-light">Step #2: Uploading</h4>
                      <p className="text-center text-white-50 pb-5 w-50 mx-auto">Government identification, personal information, and prison health records are securely uploaded to our cloud databases. These documents are also cryptographically signed and encrypted.</p>
                    </div>
                    <div className="carousel-item">
                      <img src={Access} className="d-block w-100 mx-auto my-4 d-block home-svgs" alt="..." />
                      <h4 className="text-center text-light">Step #3: Access</h4>
                      <p className="text-center text-white-50 pb-5 w-50 mx-auto">Those experiencing homelessness can access their documents at public or nonprofit computers.</p>
                    </div>
                    <div className="carousel-item">
                      <img src={FileCloud} className="d-block w-100 mx-auto my-4 d-block home-svgs" alt="..." />
                      <h4 className="text-center text-light">Step #4: Harnessing Data: Clients</h4>
                      <p className="text-center text-white-50 pb-5 w-50 mx-auto">Those experiencing homelessness (we call them clients) can now use their data to apply for jobs, print their documents, and send autofilled aid applications.</p>
                    </div>
                    <div className="carousel-item">
                      <img src={Spreadsheet} className="d-block w-100 mx-auto my-4 d-block home-svgs" alt="..." />
                      <h4 className="text-center text-light">Step #5: Harnessing Data: Nonprofits</h4>
                      <p className="text-center text-white-50 pb-5 w-50 mx-auto">Nonprofits can utilize data to generate reports, create additional touch points for care, and streamline their filing operations.</p>
                    </div>
                  </div>
                  <a className="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-slide="prev">
                    <span className="carousel-control-prev-icon" aria-hidden="true" />
                    <span className="sr-only">Previous</span>
                  </a>
                  <a className="carousel-control-next" href="#carouselExampleCaptions" role="button" data-slide="next">
                    <span className="carousel-control-next-icon" aria-hidden="true" />
                    <span className="sr-only">Next</span>
                  </a>
                </div>
              </div>
            </div>

            <div className="fluid-container mb-4 pt-5 pb-5 mx-0 background">
              <div className="container">
                <div className="row">
                  {/* <div className="col-4 d-flex align-items-center">
                    <h1 className="font-weight-bold text-center">We understand the need for secure and convenient document access.</h1>
                  </div> */}
                  <div className="col">
                    <div className="row">
                      <div className="col text-center d-flex justify-content-center">
                        <h1 className="font-weight-bold">Our Core User Values:</h1>
                      </div>
                    </div>
                    <div className="row mt-4">
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <img alt="security" src={SecureGraphic} className="home-svgs" />
                      </div>
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <img alt="security" src={ConstantAvailabilityGraphic} className="home-svgs" />
                      </div>
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <img alt="security" src={DocTransferGraphic} className="home-svgs" />
                      </div>
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <img alt="security" src={ControlGraphic} className="home-svgs" />
                      </div>
                    </div>
                    <div className="row">
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <h3 className="text-center"> Secure Access</h3>
                      </div>
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <h3 className="text-center"> Constant Availability</h3>
                      </div>
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <h3 className="text-center"> Protected Document Transfer</h3>
                      </div>
                      <div className="col-3 mt-4 d-flex align-items-center justify-content-center">
                        <h3 className="text-center"> Ease of Use</h3>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div className="container my-4 py-5">
              <div className="row">
                <h1 className="font-weight-bold">Benefiting Both Local Non-Profits and the Homeless</h1>
              </div>
              <div className="row mt-4">
                <div className="col-6">
                  <h3>Local nonprofits achieve exponentially faster service times</h3>
                  <ul className="pl-4 mt-2">
                    <li className="home-text">Stronger client relationships lead to greater touch profits</li>
                    <li className="home-text">Greater audience reach</li>
                    <li className="home-text">Higher efficiency with paperwork</li>
                  </ul>
                </div>
                <div className="col-6">
                  <h3>Homeless receive vital resources</h3>
                  <ul className="pl-4 mt-2">
                    <li className="home-text">Safe virtual document storage</li>
                    <li className="home-text">Ease of applying for jobs and government aid</li>
                    <li className="home-text">Control over personal data</li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default Home;
