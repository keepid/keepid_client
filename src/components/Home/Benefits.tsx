import React from 'react';
import { defineMessages, useIntl } from 'react-intl';

import Building from '../../static/images/building.svg';
import Profile from '../../static/images/profile-pic.svg';

const benefitsMessages = defineMessages({
  header: {
    id: 'home.benefits.header',
    defaultMessage: 'Benefiting both local non-profits and the homeless',
  },

  localHeader: {
    id: 'home.benefits.local.header',
    defaultMessage: 'Local nonprofits achieve exponentially faster service times',
  },

  localDetail1: {
    id: 'home.benefits.local.detail-1',
    defaultMessage: 'Stronger client relationships lead to greater touch points',
  },
  localDetail2: {
    id: 'home.benefits.local.detail-2',
    defaultMessage: 'Greater audience reach',
  },
  localDetail3: {
    id: 'home.benefits.local.detail-3',
    defaultMessage: 'Higher efficiency with paperwork',
  },

  homelessHeader: {
    id: 'home.benefits.homeless.header',
    defaultMessage: 'Homeless receive vital resources',
  },

  homelessDetail1: {
    id: 'home.benefits.homeless.detail-1',
    defaultMessage: 'Safe virtual document storage',
  },
  homelessDetail2: {
    id: 'home.benefits.homeless.detail-2',
    defaultMessage: 'Ease of applying for jobs and government aid',
  },
  homelessDetail3: {
    id: 'home.benefits.homeless.detail-3',
    defaultMessage: 'Control over personal data',
  },
});

const Benefits = () => {
  const intl = useIntl();

  return (
    <div className="row">
      <div className="container py-2 my-auto">
        <div className="row mt-4 mx-4 text-center d-flex justify-content-center">
          <h1 className="font-weight-bold">{intl.formatMessage(benefitsMessages.header)}</h1>
        </div>
      </div>
      <div className="container py-2 my-auto">
        <div className="row mt-4 mx-4 align-items-center">
          <div className="col-lg-5">
            <img className="img-fluid rounded mb-4 mb-lg-0" src={Building} alt="building" />
          </div>
          <div className="col-lg-7">
            <h3>{intl.formatMessage(benefitsMessages.localHeader)}</h3>
            <ul className="pl-4 mt-2">
              <li className="home-text">{intl.formatMessage(benefitsMessages.localDetail1)}</li>
              <li className="home-text">{intl.formatMessage(benefitsMessages.localDetail2)}</li>
              <li className="home-text">{intl.formatMessage(benefitsMessages.localDetail3)}</li>
            </ul>
          </div>
        </div>
      </div>
      <div className="container py-2 my-auto">
        <div className="row mt-4 mx-4 align-items-center">
          <div className="col-lg-5 d-flex content-justify-center">
            <img className="img-fluid rounded mx-auto mb-4 " src={Profile} alt="profile pic" />
          </div>
          <div className="col-lg-7">
            <h3>{intl.formatMessage(benefitsMessages.homelessHeader)}</h3>
            <ul className="pl-4 mt-2">
              <li className="home-text">{intl.formatMessage(benefitsMessages.homelessDetail1)}</li>
              <li className="home-text">{intl.formatMessage(benefitsMessages.homelessDetail2)}</li>
              <li className="home-text">{intl.formatMessage(benefitsMessages.homelessDetail3)}</li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Benefits;
