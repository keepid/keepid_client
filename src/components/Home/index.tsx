import React from 'react';
import { Helmet } from 'react-helmet';
import { defineMessages, useIntl } from 'react-intl';
import { Link } from 'react-router-dom';

import HomepageGraphic from '../../static/images/homepage_graphic.svg';
import EnsuringSecurity from './EnsuringSecurity';
import FocusingOnPeople from './FocusingOnPeople';
import HomelessStats from './HomelessStats';
import IntroText from './IntroText';
import WhoWeAreServing from './WhoWeAreServing';

const messages = defineMessages({
  title: {
    id: 'home.title',
    defaultMessage: 'Welcome',
    description: 'Home welcome message',
  },
  brandHeader: {
    id: 'home.brand-header',
    defaultMessage: 'An identification platform',
    description: 'The tagline shown on the home page',
  },
  subHeader: {
    id: 'home.sub-header',
    defaultMessage: 'for those experiencing homelessness',
  },
  getStarted: { id: 'home.get-started', defaultMessage: 'Get Started' },
  learnMore: { id: 'home.learn-more', defaultMessage: 'Donate' },
});

function Home() {
  const intl = useIntl();

  return (
    <div>
      <Helmet>
        <title>{intl.formatMessage(messages.title)}</title>
        <meta name="description" content="Keep.id" />
      </Helmet>
      {/* <StatementOfSolidarity /> */}
      <div className="container-fluid my-auto">
        <div className="d-flex flex-row section1 mt-5 justify-content-center img-fluid">
          <div className="flex-col px-4 custom-vertical-center">
            <div className="rounded mb-3 pb-5">
              <div className="page-header">
                <span className="brand-text" id="brand-header">
                  {intl.formatMessage(messages.brandHeader)}
                </span>
              </div>
              <p className="pt-2 pb-2 home-subtext">
                {intl.formatMessage(messages.subHeader)}
              </p>
              <Link to="/signup-branch">
                <button
                  type="button"
                  className="btn btn-secondary btn-lg w-40 mr-2 mb-2"
                >
                  {intl.formatMessage(messages.getStarted)}
                </button>
              </Link>
              <a
                href="https://team.keep.id"
                type="button"
                className="btn btn-outline-secondary btn-lg w-40 mr-2 mb-2"
              >
                {intl.formatMessage(messages.learnMore)}
              </a>
            </div>
          </div>
          <div className="flex-col px-5 custom-vertical-center">
            <div className="pb-4 container-home-right">
              <div>
                <img
                  alt="Document Platform"
                  src={HomepageGraphic}
                  className="home-form-svg text-left"
                />
              </div>
            </div>
          </div>
        </div>
      </div>

      <IntroText />

      <FocusingOnPeople />

      <EnsuringSecurity />

      <HomelessStats />

      <WhoWeAreServing />
    </div>
  );
}

export default Home;
