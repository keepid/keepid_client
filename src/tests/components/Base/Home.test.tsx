import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import userEvent from '@testing-library/user-event';
import { MemoryRouter } from 'react-router-dom';
import Home from '../../../components/Base/Home';

window.scroll = jest.fn();
test('Home page loads', () => {
  const { getByText } = render(<MemoryRouter><Home /></MemoryRouter>);
  getByText('Safeguarding identities of those experiencing homelessness');
});

test('Home page buttons work', () => {
  const { getByText } = render(<MemoryRouter><Home /></MemoryRouter>);
  const button = getByText('Get Started');
  userEvent.click(button);
  const button2 = getByText('Learn More');
  userEvent.click(button2);
});

test('Home page snapshot test', () => {
  const tree = renderer.create(<MemoryRouter><Home /></MemoryRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
