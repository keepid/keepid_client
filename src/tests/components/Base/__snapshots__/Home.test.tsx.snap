// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page snapshot test 1`] = `
<div>
  <div>
    <div
      className="w-100 statement-solidarity"
    >
      <button
        aria-label="Close"
        className="close float-right mr-5 mt-1"
        onClick={[Function]}
        type="button"
      >
        <span
          aria-hidden="true"
        >
          Ã—
        </span>
      </button>
      <div
        className="container text-center py-2 font-weight-bold"
      >
        We condemn the rising wave of violence and xenophobic rhetoric against members of the Asian American Pacific Islander (AAPI) community. Read our Statement of Solidarity with the AAPI Community 
        <a
          href="Team Keep AAPI Solidarity Statement.pdf"
          rel="noreferrer"
          target="_blank"
        >
          here
        </a>
      </div>
    </div>
  </div>
  <div
    className="container-fluid my-auto"
  >
    <div
      className="row section1 mt-5 justify-content-center"
    >
      <div
        className="col-md-5 px-4 custom-vertical-center"
      >
        <div
          className="rounded mb-3 pb-5"
        >
          <div
            className="page-header"
          >
            <span
              className="brand-text"
              id="brand-header"
            >
              An identification platform
            </span>
          </div>
          <p
            className="pt-2 pb-2 home-subtext"
          >
            for those experiencing homelessness
          </p>
          <a
            href="/signup-branch"
            onClick={[Function]}
          >
            <button
              className="btn btn-secondary btn-lg w-40 mr-2 mb-2"
              type="button"
            >
              Get Started
            </button>
          </a>
          <a
            className="btn btn-outline-secondary btn-lg w-40 mr-2 mb-2"
            href="https://team.keep.id"
            type="button"
          >
            Donate
          </a>
        </div>
      </div>
      <div
        className="col-md-5 px-5 custom-vertical-center"
      >
        <div
          className="pb-4 container-home-right"
        >
          <div>
            <img
              alt="Document Platform"
              className="home-form-svg text-left"
              src="homepage_graphic.svg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="container py-5"
  >
    <div
      className="row viewport-height-50"
    >
      <div
        className="col-md-10 custom-vertical-center mx-auto text-center"
      >
        <div
          id="info"
        >
          <img
            alt="rectangle"
            className="svg-purple mb-5"
            src="rectangle.svg"
          />
          <h3
            className="hero-subtext text-grey font-weight-light pb-3"
          >
            People experiencing homelessness need a platform to assist in applying for, securely storing, and utilizing ID to access services.
          </h3>
          <h3
            className="hero-subtext text-black font-weight-medium pb-3"
          >
            We provide a guided application process integrated with non-profit networks and HIPAA-compliant ID storage, designed for those with low technological literacy.
          </h3>
          <a
            href="#info"
            onClick={[Function]}
          >
            <button
              className="btn btn-outline-secondary btn-lg w-40"
              type="button"
            >
              Learn More
            </button>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-5"
  >
    <div
      className="w-100 partial-background"
    >
      <div
        className="container"
      >
        <h1
          className="text-white pt-4 pb-4"
        >
          Focusing on People
        </h1>
      </div>
    </div>
    <div
      className="container my-5"
    >
      <div
        className="row"
      >
        <div
          className="col-md-6 custom-vertical-center px-4"
        >
          <div>
            <h3>
              Create an Account
            </h3>
            <p
              className="pt-2 pb-2 home-paragraph-text"
            >
              Individuals experiencing homelessness (we call them clients)
                create a Keep.id account at participating nonprofits. Nonprofits
                then help homeless obtain missing identification.
            </p>
          </div>
        </div>
        <div
          className="col-md-6 p-4"
        >
          <img
            alt="Create an account image"
            className="home-form-svg text-left"
            src="create_account.png"
          />
        </div>
      </div>
    </div>
    <div
      className="container my-5"
    >
      <div
        className="row"
      >
        <div
          className="col-md-6 p-4 d-none d-md-block"
        >
          <img
            alt="Create an account image"
            className="home-form-svg text-left"
            src="my_documents.png"
          />
        </div>
        <div
          className="col-md-6 custom-vertical-center px-4"
        >
          <div>
            <h3>
              Upload Documents
            </h3>
            <p
              className="pt-2 pb-2 home-paragraph-text"
            >
              Government identification, personal information, and prison
                health records are securely uploaded to our cloud databases.
                These documents are also cryptographically signed and encrypted.
            </p>
          </div>
        </div>
        <div
          className="col-md-6 p-4 d-md-none"
        >
          <img
            alt="Create an account image"
            className="home-form-svg text-left"
            src="my_documents.png"
          />
        </div>
      </div>
    </div>
    <div
      className="container my-5"
    >
      <div
        className="row"
      >
        <div
          className="col-md-6 custom-vertical-center px-4"
        >
          <div>
            <h3>
              Access Documents
            </h3>
            <p
              className="pt-2 pb-2 home-paragraph-text"
            >
              Those experiencing homelessness can access their documents at
                public or nonprofit computers.
            </p>
          </div>
        </div>
        <div
          className="col-md-6 p-4"
        >
          <img
            alt="Create an account image"
            className="home-form-svg text-left"
            src="access_documents.png"
          />
        </div>
      </div>
    </div>
    <div
      className="container my-5"
    >
      <div
        className="row"
      >
        <div
          className="col-md-6 p-4 d-none d-md-block"
        >
          <img
            alt="Create an account image"
            className="home-form-svg text-left"
            src="streamline_applications.png"
          />
        </div>
        <div
          className="col-md-6 custom-vertical-center px-4"
        >
          <div>
            <h3>
              Streamline Applications
            </h3>
            <p
              className="pt-2 pb-2 home-paragraph-text"
            >
              Those experiencing homelessness can now use their data to apply
                for jobs, print their documents, and send autofilled aid
                applications.
            </p>
            <p
              className="pt-2 pb-2 home-paragraph-text"
            >
              Nonprofits can utilize data to generate reports, create
                additional touch points for care, and streamline their filing
                operations.
            </p>
            <a
              href="#info"
              onClick={[Function]}
            >
              <button
                className="btn btn-outline-secondary btn-lg w-40 mr-2 mb-2"
                type="button"
              >
                Learn More
              </button>
            </a>
          </div>
        </div>
        <div
          className="col-md-6 p-4 d-md-none"
        >
          <img
            alt="Create an account image"
            className="home-form-svg text-left"
            src="streamline_applications.png"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-5"
  >
    <div
      className="w-100 partial-background"
    >
      <div
        className="container"
      >
        <h1
          className="text-white pt-4 pb-4"
        >
          Ensuring Security
        </h1>
      </div>
    </div>
    <div
      className="container my-5"
    >
      <div
        className="row"
      >
        <div
          className="col-md-4"
        >
          <div
            className="ml-1 mr-1 mt-2 mb-2"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="border-0 shadow card"
              style={
                Object {
                  "borderRadius": 20,
                  "height": "485px",
                  "maxWidth": "325px",
                  "width": "auto",
                }
              }
            >
              <div
                className="p-0 d-flex flex-column align-items-start card-body"
                style={
                  Object {
                    "height": "100%",
                  }
                }
              >
                <img
                  alt="..."
                  className=""
                  src="HIPAA_badge.png"
                  style={
                    Object {
                      "borderTopLeftRadius": 20,
                      "borderTopRightRadius": 20,
                      "height": "50%",
                      "objectFit": "cover",
                      "overflow": "hidden",
                      "width": "100%",
                    }
                  }
                />
                <div
                  className="p-4"
                >
                  <div
                    className="card-title h5"
                  >
                    <h3>
                      HIPAA Compliant
                    </h3>
                  </div>
                  <p
                    className="card-text"
                  >
                    Team Keep members are certified and follow "national standards for 
      electronic health care transactions and code sets, 
      unique health identifiers, and security."
                  </p>
                  <div>
                    <a
                      className="card-link"
                      href="https://compliancy-group.com/"
                    >
                      Learn More
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-md-4"
        >
          <div
            className="ml-1 mr-1 mt-2 mb-2"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="border-0 shadow card"
              style={
                Object {
                  "borderRadius": 20,
                  "height": "485px",
                  "maxWidth": "325px",
                  "width": "auto",
                }
              }
            >
              <div
                className="p-0 d-flex flex-column align-items-start card-body"
                style={
                  Object {
                    "height": "100%",
                  }
                }
              >
                <img
                  alt="..."
                  className=""
                  src="googletink.png"
                  style={
                    Object {
                      "borderTopLeftRadius": 20,
                      "borderTopRightRadius": 20,
                      "height": "50%",
                      "objectFit": "cover",
                      "overflow": "hidden",
                      "width": "100%",
                    }
                  }
                />
                <div
                  className="p-4"
                >
                  <div
                    className="card-title h5"
                  >
                    <h3>
                      Always Encryped
                    </h3>
                  </div>
                  <p
                    className="card-text"
                  >
                    We use state-of-the-art security practices and 
    libraries used by industry leaders, such as Google. 
    Your documents are cryptographically signed and encrypted to increase client security.
                  </p>
                  <div>
                    <a
                      className="card-link"
                      href="https://developers.google.com/tink"
                    >
                      Learn More
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-md-4"
        >
          <div
            className="ml-1 mr-1 mt-2 mb-2"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <div
              className="border-0 shadow card"
              style={
                Object {
                  "borderRadius": 20,
                  "height": "485px",
                  "maxWidth": "325px",
                  "width": "auto",
                }
              }
            >
              <div
                className="p-0 d-flex flex-column align-items-start card-body"
                style={
                  Object {
                    "height": "100%",
                  }
                }
              >
                <img
                  alt="..."
                  className=""
                  src="mozilla_observatory.png"
                  style={
                    Object {
                      "borderTopLeftRadius": 20,
                      "borderTopRightRadius": 20,
                      "height": "50%",
                      "objectFit": "cover",
                      "overflow": "hidden",
                      "width": "100%",
                    }
                  }
                />
                <div
                  className="p-4"
                >
                  <div
                    className="card-title h5"
                  >
                    <h3>
                      Secure in the Cloud
                    </h3>
                  </div>
                  <p
                    className="card-text"
                  >
                    Our application has been validated against internet vulnerabilities 
    by Mozilla Observatory. We hold ourselves to the highest web industry standards.
                  </p>
                  <div>
                    <a
                      className="card-link"
                      href="https://observatory.mozilla.org/analyze/keep.id"
                    >
                      Learn More
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="w-100 partial-background"
  >
    <div
      className="container"
    >
      <h1
        className="text-white pt-4 pb-4"
      >
        A Systemic Cycle
      </h1>
    </div>
  </div>
  <div
    className="container mt-4 mb-4 pt-5 pb-5"
  >
    <div
      className="row d-flex align-items-center"
      id="info"
    >
      <div
        className="col-md-6 "
      >
        <h1
          className="text-center m-3 pb-5"
        >
          The lack of 
          <span
            className="text-primary-theme font-weight-bold"
          >
            identification
          </span>
           is a serial struggle for those experiencing homelessness.
        </h1>
      </div>
      <div
        className="col-md-6"
      >
        <div
          className="row d-flex align-items-center"
        >
          <div
            className="col-6 text-center"
          >
            <h1
              className="font-weight-bold statistic-text"
            >
              54%
            </h1>
          </div>
          <div
            className="col-6"
          >
            <span
              className="statistic-subtext"
            >
              Denied access to shelters or adequate housing services
            </span>
          </div>
        </div>
        <div
          className="row d-flex align-items-center"
        >
          <div
            className="col-6 text-center"
          >
            <h1
              className="font-weight-bold statistic-text"
            >
              53%
            </h1>
          </div>
          <div
            className="col-6"
          >
            <span
              className="statistic-subtext"
            >
              Denied access to food stamps
            </span>
          </div>
        </div>
        <div
          className="row d-flex align-items-center"
        >
          <div
            className="col-6 text-center"
          >
            <h1
              className="font-weight-bold statistic-text"
            >
              45%
            </h1>
          </div>
          <div
            className="col-6"
          >
            <span
              className="statistic-subtext"
            >
              Denied access to Medicaid or other medical services
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="w-100 partial-background"
  >
    <div
      className="container"
    >
      <h1
        className="text-white pt-4 pb-4"
      >
        Our Partners
      </h1>
    </div>
  </div>
  <div
    className="container-fluid mx-0 pt-5 pb-4"
  >
    <div
      className="container section2"
    >
      <div
        className="row text-center"
      >
        <div
          className="col-md-4 p-4"
        >
          <img
            alt="Why Not Prosper"
            className="home-form-svg text-left p-4"
            src="whynotprosper.png"
          />
        </div>
        <div
          className="col-md-4"
        >
          <img
            alt="Roxborough Food Pantry"
            className="home-form-svg text-left p-5"
            src="roxborough.png"
          />
        </div>
        <div
          className="col-md-4"
        >
          <img
            alt="Venturelab"
            className="home-form-svg text-left"
            src="venturelab.jpg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
